pennylane/io/io.py

```diff
@@ -827,3 +835,38 @@ def from_quil_file(quil_filename: str):
     """
     plugin_converter = plugin_converters["quil_file"].load()
     return plugin_converter(quil_filename)
+
+
+def from_qasm3(quantum_circuit: str):
+    """
+    Loads a simple QASM 3.0 quantum circuits involving basic usage of gates from a QASM string using the QASM
```

Comment:
Is it possible to list OpenQASM3 gates that aren't supported in the doc string? Maybe a `.. note::` before Args and Returns